velox/exec/Aggregate.h

```diff
@@ -241,6 +270,11 @@ class Aggregate {
   // Offset of fixed length accumulator state in group row.
   int32_t offset_;
 
+  // Offset of uint32_t row byte size of row. 0 if accumulator is
+  // fixed width. The size is capped at 4G and will stay at 4G and not
```

Comment:
Can you explain the motivation for 4G cap and why it works to truncate past that?